runtime/manual/basics/modules/remote_modules.md

```diff
@@ -0,0 +1,165 @@
+# Using Remote Modules with Deno
```

Comment:
"Remote Modules" is not the most widely understandable term - consider changing the title of this section.

Maybe "Using third party modules" or "Using external code" or "Using libraries"

Reply:
Yeah, thinking about it, I agree, this is going to be the primary usecase. I'll move it into index.